.github/workflows/ci.yml

```diff
@@ -58,3 +59,62 @@ jobs:
             -
                 name: "Run friendsofphp/php-cs-fixer"
                 run: "vendor/bin/php-cs-fixer fix --dry-run --diff"
+
+    test:
+        name: "Test PHP ${{ matrix.php-version }} + maker-bundle version: ${{ matrix.maker-test-version }}"
+
+        runs-on: ubuntu-latest
+
+        continue-on-error: ${{ matrix.allow-failures }}
+
+        strategy:
+            matrix:
+                php-version:
+                    - '7.1.33'
+                    - '7.3'
+                    - '7.4'
+                maker-test-version:
+                    - 'stable'
+                allow-failures: [false]
+                include:
+                    - php-version: '7.4'
+                      maker-test-version: 'dev'
+                      allow-failures: true
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2.3.3
+
+            - name: Install PHP with extensions
+              uses: shivammathur/setup-php@2.7.0
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  tools: composer:v2
+
+            - name: Add PHPUnit matcher
+              run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+            - name: Set Composer cache directory
+              id: composer-cache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache Composer
+              uses: actions/cache@v2.1.2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-${{ matrix.php-version }}-composer-${{ matrix.dependencies }}-${{ matrix.variant }}
+                  restore-keys: ${{ runner.os }}-${{ matrix.php-version }}-composer-
+
+#            - name: Require symfony/flex
+#              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex dev-master
```

Comment:
```suggestion
            - name: Require symfony/flex
              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex dev-main
```

Reply:
Thanks, I will fix Travis in a separate PR 👍